indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -186,9 +186,11 @@
                     {% endif %}
                 </div>
                 <div>
-                    {% trans url=url_for('contributions.display_contribution', abstract.contribution) %}
-                        <a href="{{ url }}">Go to contribution</a>
-                    {% endtrans %}
+                    {% if abstract.contribution %}
+                        {% trans url=url_for('contributions.display_contribution', abstract.contribution) %}
```

Comment:
The trans tag has inner whitespace that's not stripped.

Anyway, I'd move the link outside the trans block.